rules/truenas-community-edition.rules

```diff
@@ -0,0 +1,10 @@
+## These rules are to be applied to TrueNAS Community Edition
+##
+## These rules quiet the PAM logging
+-A exclude,always -F msgtype=USER_AUTH
+-A exclude,always -F msgtype=USER_START
+-A exclude,always -F msgtype=USER_END
+-A exclude,always -F msgtype=USER_ACCT
+-A exclude,always -F msgtype=CRED_REFR
+-A exclude,always -F msgtype=CRED_DISP
+-A exclude,always -F msgtype=CRED_ACQ
```

Comment:
Let's add some rules to.
1. generate entries for `incus` commands where auid is not unset
2. generate entries for access to `/data/` where auid is not unset (manual edits to sensitive data) -- copy line from STIG